.travis.yml

```diff
@@ -12,15 +11,6 @@ branches:
   except:
     - gh-pages
 
-# addons:
```

Comment:
not to be too nitpicky, but these are useful comments for me considering Travis is currently in the final stages of approving my request to whitelist the qpid-testing apt-source